<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/diagram_64.png</normaloff>:/img/diagram_64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="verticalTabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sampleTab2">
       <attribute name="icon">
        <iconset>
         <normaloff>:/ui/img/briefcase_64.png</normaloff>:/ui/img/briefcase_64.png</iconset>
       </attribute>
       <attribute name="title">
        <string>sampleTabName</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,2,0,2">
        <item>
         <widget class="QGroupBox" name="hGBoxKitName">
          <layout class="QHBoxLayout" name="hLayoutName" stretch="1,5">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="nameKitLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>Kit name :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nameKitName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>nameNameName</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="hGBoxPathMt4">
          <layout class="QHBoxLayout" name="hLayoutPath" stretch="1,2,1,2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="serverLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Server :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="serverName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>server.ru/server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pathToMt4Label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string>Path to MT4 :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pathToMt4Name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>path:pathpath</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLayoutConf">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="vGBoxInput">
            <property name="title">
             <string>Input neural network :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="vLayoutInput">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QListView" name="inputListView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hLayoutInputSize" stretch="1,2">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="inputLabel">
                 <property name="text">
                  <string>History size :</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="inputSize">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vLayoutSymbol" stretch="2,1,2">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="arrowLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/img/right_64.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::NoTextInteraction</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="vGBoxOutput">
            <property name="title">
             <string>Output neural network :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="vLayoutOutput">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QListView" name="outputListView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hLayoutOutputSize" stretch="1,2">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="outputLabel">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Prediction size :</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="outputSize">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vLayoutButtons">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="configurationButton">
              <property name="text">
               <string>Configuration</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/ui/img/gear_64.png</normaloff>:/ui/img/gear_64.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="format">
           <string>Progress %p%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="consoleTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_New_Kit"/>
    <addaction name="actionOpen_Kit"/>
    <addaction name="action_Save_Kit"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionKit_Configuration"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_New_Kit"/>
   <addaction name="actionOpen_Kit"/>
   <addaction name="action_Save_Kit"/>
   <addaction name="separator"/>
   <addaction name="actionTrain_NN"/>
   <addaction name="actionStart_forecasting"/>
   <addaction name="actionStop"/>
   <addaction name="actionKit_Configuration"/>
  </widget>
  <action name="actionAdd_New_Kit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/document_64.png</normaloff>:/img/document_64.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;New Kit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/gear_64.png</normaloff>:/img/gear_64.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_Save_Kit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/save_64.png</normaloff>:/img/save_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Kit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete_Kit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/delete_64.png</normaloff>:/img/delete_64.png</iconset>
   </property>
   <property name="text">
    <string>Delete Kit</string>
   </property>
  </action>
  <action name="actionOpen_Kit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/folder_64.png</normaloff>:/img/folder_64.png</iconset>
   </property>
   <property name="text">
    <string>Open Kit</string>
   </property>
  </action>
  <action name="actionTrain_NN">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/clipboard_pencil_64.png</normaloff>:/img/clipboard_pencil_64.png</iconset>
   </property>
   <property name="text">
    <string>Train NN</string>
   </property>
   <property name="toolTip">
    <string>Train Neural Network</string>
   </property>
  </action>
  <action name="actionStart_forecasting">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/statistics2_diagram_64.png</normaloff>:/img/statistics2_diagram_64.png</iconset>
   </property>
   <property name="text">
    <string>Start forecasting</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/stop_64.png</normaloff>:/img/stop_64.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop work</string>
   </property>
  </action>
  <action name="actionKit_Configuration">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/gear_64.png</normaloff>:/img/gear_64.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
